Doc/source/designspaceLib/xml.rst

```diff
@@ -1012,3 +1040,16 @@ The ``<lib>`` element contains arbitrary data.
     </lib>
 
 
+Type equivalence (`plist`)
+==========================
```

Comment:
Why did you include this section? I don't think it's necessary, because most of the Designspace XML doesn't use plist (instead defines its own tags and attributes). At most it embeds plist for the lib sections, but the contents of these lib keys is not really defined in this document.

Reply:
I included this section due to `org.statmake.stylespace`. As in https://github.com/fonttools/fonttools/pull/2632#discussion_r886616940, the section comes to us due to a line in MFEK Serif's designspace:

https://github.com/indestructible-type/Besley/blob/c87ddc8b1d852033865cb52680b6bddb3da1b943/sources/designspace/Besley.designspace#L263

I strongly suggest nipping this in the bud now as `stylespace` isn't going anywhere.

Reply:
I would consider that the typing of the contents of a lib key (such as the stylespace lib key) is out of scope of the designspace documentation, because these lib keys could contain anything at all. In this particular case stylespace might sound related to the DS5 but as I say it's "just one of the lib keys" and as you say it's meant to be deprecated in favour of the DS5 additions around STAT anyway, so I think the section can go.

Reply:
Oh, OK. If indeed `org.statmake.stylespace` _is_ going somewhere than you're right, the section probably serves no purpose. :-)